app/requirements.txt

```diff
@@ -5,4 +5,5 @@ itsdangerous==0.24
 jinja2==2.9.6
 markupsafe==1.0
 werkzeug==0.12.2
-wtforms==2.1
\ No newline at end of file
+wtforms==2.1
+pyflakes==1.6.0
```

Comment:
pyflakes is only required for development, not on a running server, it could go into the file dev-requirements.txt